OpenRA.Game/Graphics/ChromeProvider.cs

```diff
@@ -55,35 +88,29 @@ public static void Deinitialize()
 			cachedSprites = null;
```

Comment:
We need to null `cachedPanelSprites` here too!

Reply:
Is this just a "note to self for future PR" kind of thing, or does this PR itself need an update itself?